Timer.DAL/Timer.DAL/Repositories/UserRepository.cs

```diff
@@ -172,5 +172,30 @@ public void Save()
         {
             timerContext.SaveChanges();
         }
+
+        public Task SetPasswordHashAsync(User user, string passwordHash)
```

Comment:
wait a minute.. why do we need this methods again? where are they used right now?

Reply:
just for the future use?

Reply:
Yeah, you, right, we don`t explicitly use this three method, but it is used in createAsync method, is obligated inherite IUserPasswordStore interface

Reply:
one minute, I make a screenshot error

Reply:
![image](https://user-images.githubusercontent.com/37620894/50514985-8266bd00-0aaa-11e9-9657-22f622b34c07.png)

Reply:
btw, iUserPasswordStore implements IUserStore. It's unnecessary to have both of them on UserRepository
![image](https://user-images.githubusercontent.com/45561295/50514989-8c88bb80-0aaa-11e9-927f-d978e1e248f1.png)

Reply:
This error when I try add new user to database

Reply:
Okey, then delete IUserStore

Reply:
got it, just remove unnecessary IUserStore from Userrepository. Then we have 3 interfaces on that class which is not that bad (+ I don't want to hide IRepository under other interface). So at the end we can leave it like this:
public class UserRepository : IUserEmailStore<User, int>, IRepository<User>, IQueryableUserStore<User, int>, IUserPasswordStore<User, int>

without specifying additional interface

Reply:
Ok, will change

Reply:
Can I merge this code to develop branch?